ssl/ssl_lib.c

```diff
@@ -2552,23 +2552,32 @@ int SSL_set_cipher_list(SSL *s, const char *str)
 char *SSL_get_shared_ciphers(const SSL *s, char *buf, int len)
```

Comment:
Could I ask that `len` be renamed to `size`?  The reason is that there's a common pattern where `len` indicates that `buf` is an input buffer with `len` chars of content, while `size` indicates that `buf` is an output buffer with a maximum size.  So `size` makes more sense, intuitively.

Reply:
Done. I also made the same change in the other two PRs (#6114 and #6115)